### ISSUE TITLE
failwith/failwithf warning support

### ISSUE THREAD

**Issue by bleis-tift** [fsprojects/FSharpLint#34]
feature request.

``` fsharp
failwith "%s" str // typo of failwith*f*. but this code has no warnings...
failwithf "%s %s" str1 str2 str3 // %s is not enough or str3 is extra.
```

I want to detect these cases.
